src/install/config/python.yaml

```diff
@@ -0,0 +1,750 @@
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-scgi-web-server
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-starlette-web-framework
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-starlette-web-framework
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-tornado-6-web-framework
+
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-tornado-3
+  - /node/1971
+  - /docs/python/python-agent-and-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-and-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/nr-only-python-agent-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-tornado-experimental
+  - /docs/agents/python-agent/hosting-mechanisms/introductory-tornado-4-support
+  - /docs/agents/python-agent/web-frameworks-servers/using-trace-decorators-tornado-coroutines
+  - /docs/agents/python-agent/trace-decorators-tornado-coroutines
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-tornado-6-web-framework
+
+  - /docs/agents/python-agent/web-frameworks-servers/uvicorn
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-uvicorn-web-server
+  - /docs/agents/python-agent/web-frameworks-servers/uvicorn-web-server
+  - /docs/apm/agents/python-agent/web-frameworks-servers/uvicorn
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-uwsgi-web-server
+  - /docs/python/python-agent-and-uwsgi
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-and-uwsgi
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-uwsgi
+  - /docs/apm/agents/python-agent/web-frameworks-servers
+  - /docs/agents/python-agent/web-frameworks-and-servers
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-uwsgi-web-server
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-waitress-web-server
+  - /docs/python/python-agent-and-waitress
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-and-waitress
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-waitress
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-waitress-web-server
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-web2py-web-framework
+  - /docs/python/python-agent-and-web2py
+  - /docs/agents/python-agent/web-frameworks/python-agent-and-web2py
+  - /docs/agents/python-agent/web-frameworks/python-agent-web2py
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-web2py-web-framework
+
+
+# -----------------------------------------------------------------
+# Define the categories for each type of question you'll be asking:
+# -----------------------------------------------------------------
+appInfo:
+
+
+
+  - optionType: python-non-web
+    label: "We'll show you how to instrument your non-web Python app"
```

Comment:
In general, I'm finding these labels not very helpful. For example, this label is just restating the question in step 1. Is there a way this label can provide useful context? If not, I'd recommend just deleting it.

Reply:
@paperclypse As I mentioned below, the label field is required. I'm not sure how to get around this.

Reply:
@paperclypse I just learned from a colleague that we can just have an empty value between single quotes. So, I'll remove these. Previously, I had tested removing the entire field, and that wrecks the build.